src/main/java/org/broadinstitute/hellbender/tools/variantdb/nextgen/ExtractCohortRecord.java

```diff
@@ -0,0 +1,119 @@
+package org.broadinstitute.hellbender.tools.variantdb.nextgen;
+
+import htsjdk.samtools.util.Locatable;
+import org.apache.avro.generic.GenericRecord;
+import org.broadinstitute.hellbender.tools.variantdb.SchemaUtils;
+
+import java.util.Objects;
+
+public class ExtractCohortRecord implements Locatable {
+
+    private final Long location;
+    private final String sampleName;
+    private final String contig;
+    private final Integer start;
+    private final Integer end;
+
+    private final String state;
+    private final String refAllele;
+    private final String altAllele;
+    private final String callGT;
+    private final String callGQ;
+    private final String callRGQ;
+    private final String qualapprox;
+    private final String asQualapprox;
+    private final String callPL;
+
+    // COHORT_FIELDS
+//    public static final ImmutableSet<String> COHORT_FIELDS = ImmutableSet.of(
+//            SchemaUtils.LOCATION_FIELD_NAME,
+//            SchemaUtils.SAMPLE_NAME_FIELD_NAME,
+//            SchemaUtils.STATE_FIELD_NAME,
+//            SchemaUtils.REF_ALLELE_FIELD_NAME,
+//            SchemaUtils.ALT_ALLELE_FIELD_NAME,
+//            SchemaUtils.CALL_GT,
+//            SchemaUtils.CALL_GQ,
+//            SchemaUtils.CALL_RGQ,
+//            SchemaUtils.QUALapprox,
+//            SchemaUtils.AS_QUALapprox,
+//            SchemaUtils.CALL_PL);//, AS_VarDP);
+
+
+    public ExtractCohortRecord(GenericRecord genericRecord) {
```

Comment:
ToL: same comment as constructor for `ExtractCohortFilterRecord`

Reply:
we concluded we can add this in future if it's needed